Common: ESE splines for FT0-A and FV0-A

joachimckh opened the pull request:
Added existing Q-vector framework. FFitWeights now holds the spline functionality of the event shape engineering procedure. Table columns for FT0-A and FV0-A added.

ddobrigk commented:
Thanks a lot @joachimckh ! Maybe @strogolo can take a quick look before I approve, as he is following the Q-vector-related developments more closely? Thank you!

strogolo commented:
Hi @ddobrigk,
sorry but I missed the ping of this message. @joachimckh have you tested the functionality of this PR locally? Is it working properly? If it is fine and works perfectly offline, then we can proceed with the approval.

Regards,
Stefano

joachimckh commented:
Hi @ddobrigk & @strogolo,

Thanks for the review. I have tested it locally, and it seems to work fine. I will add the TPC as well soon. 

I modified it quite a bit to incorporate the existing QVector framework. I also added a customised Merge function to the FFitWeights class since this had previously made it crash on Hyperloop (I have tested that this works by merging output locally). 

Cheers,
Joachim
